website/project/model.py

```diff
@@ -3273,14 +3273,34 @@ def is_approved(self):
         return self.approval.is_approved if self.requires_approval else True
 
     def update_metadata(self, metadata, save=True):
-        if self.is_pending_review or self.is_approved:
-            raise NodeStateError('Cannot edit while this draft is being reviewed')
+        # TODO(barbour-em): delete or implement in schema
+        # if self.is_pending_review or self.is_approved:
+        #     raise NodeStateError('Cannot edit while this draft is being reviewed')
+
         changes = []
+        old_comment_ids = list()
```

Comment:
I would rather see this logic handled on a question-by question basis than iterating over the whole of the metadata.